ingester/ingester.py

```diff
@@ -579,6 +635,10 @@ def main():
     VERBOSE = args.verbose
 
     logger.info("Starting ingestion process...")
+    start_http_server(int(os.environ.get("PROMETHEUS_PORT", 8002)))
+    logger.info(
+        f"Prometheus metrics server started on port {os.environ.get('PROMETHEUS_PORT', 8002)}"
+    )
```

Comment:
Put this at a top-level constant, to get and save the prometheus port (defaulting to 8002) and use that constant in both lines